Makefile

```diff
@@ -17,13 +17,19 @@ build: lint
 	$(DOCKER_RUN) -v $(PWD):/home/akeneo/pim-docs/data $(DOCKER_IMAGE) cp -r /home/akeneo/pim-docs/data/design_pim/styleguide /home/akeneo/pim-docs/data/pim-docs-build/design_pim/
 	@echo "\nYou are now ready to check the documentation locally in the directory \"pim-docs-build/\" and to deploy it with \"HOSTNAME=foo.com PORT=1985 VERSION=bar make deploy\"."
 
-deploy: build
+deploy: build update-versions
 	docker run -it --rm -v $${SSH_AUTH_SOCK}:/ssh-auth.sock:ro -e SSH_AUTH_SOCK=/ssh-auth.sock -v $(PWD):/home/akeneo/pim-docs/data $(DOCKER_IMAGE) rsync -e "ssh -q -p $${PORT} -o StrictHostKeyChecking=no" -qarz --delete /home/akeneo/pim-docs/data/pim-docs-build/ akeneo@$${HOSTNAME}:/var/www/${VERSION}
-
+ #VERSION_FILE=version2.json && [ master == $CIRCLE_BRANCH ] && VERSION_LABEL=master || VERSION_LABEL=v${CIRCLE_BRANCH}  && (find  $VERSION_FILE || printf "[\n]" > $VERION_FILE) && (grep -n $CIRCLE_BRANCH $VERSION_FILE || perl -pi -e ' s/ }\n/},\n  { "label": "$VERSION_NAME", "url": "\/$CIRCLE_BRANCH\/index.html" },\n/' $VERSION_FILE )
+VERSION_FILE=version2.json && [ master == $CIRCLE_BRANCH ] && VERSION_LABEL=master || VERSION_LABEL=v${CIRCLE_BRANCH}  && (find  $VERSION_FILE 2>/dev/null || printf "[\n]\n" > $VERSION_FILE) && (grep -n $CIRCLE_BRANCH $VERSION_FILE || perl -pi -e ' s/ }\n/},\n  { "label": "$VERSION_NAME", "url": "\/$CIRCLE_BRANCH\/index.html" },\n/' $VERSION_FILE )
 lint: docker-build
 	rm -rf pim-docs-build && mkdir pim-docs-build
 	rm -rf pim-docs-lint && mkdir pim-docs-lint
 	$(DOCKER_RUN) -v $(PWD):/home/akeneo/pim-docs/data $(DOCKER_IMAGE) sphinx-build -nWT -b linkcheck /home/akeneo/pim-docs/data /home/akeneo/pim-docs/data/pim-docs-lint
 
 docker-build:
 	docker build . --tag $(DOCKER_IMAGE)
+
+update-versions:
+	docker run -it --rm -v $${SSH_AUTH_SOCK}:/ssh-auth.sock:ro -e SSH_AUTH_SOCK=/ssh-auth.sock -v $(PWD):/home/akeneo/pim-docs/data $(DOCKER_IMAGE) rsync -e "ssh -q -p $${PORT} -o StrictHostKeyChecking=no" -qarz --delete akeneo@$${HOSTNAME}:/var/www/versions.php /home/akeneo/pim-docs/data
+	$(DOCKER_RUN)  -v $(PWD):/home/akeneo/pim-docs/data $(DOCKER_IMAGE) php scripts/update-doc-versions.php $(CIRCLE_BRANCH) versions.json
+	docker run -it --rm -v $${SSH_AUTH_SOCK}:/ssh-auth.sock:ro -e SSH_AUTH_SOCK=/ssh-auth.sock -v $(PWD):/home/akeneo/pim-docs/data $(DOCKER_IMAGE) rsync -e "ssh -q -p $${PORT} -o StrictHostKeyChecking=no" -qarz --delete /home/akeneo/pim-docs/data/versions.php akeneo@$${HOSTNAME}:/var/www/
```

Comment:
I think the whole line should be put in an alias if we use it several times 

```bash
docker run -it --rm -v $${SSH_AUTH_SOCK}:/ssh-auth.sock:ro -e SSH_AUTH_SOCK=/ssh-auth.sock -v $(PWD):/home/akeneo/pim-docs/data $(DOCKER_IMAGE)
```

like we did for `$(DOCKER_RUN)`